src/main/resources/sql/projectData/readBiodataProjectData.sql

```diff
@@ -0,0 +1,16 @@
+select distinct
+    activity.data_source_id
```

Comment:
We can drop the activity.data_source_id from the selected values - it's being defaulted in the processor.